### ISSUE TITLE
[Installation] Allow installation via `composer create-project`

### ISSUE THREAD

**Issue by DivineOmega** [langleyfoxall/laravel-boilerplate#6]
If we change the `composer.json` package name to `langleyfoxall/laravel-boilerplate`, and set the package up on Packagist, we should then be able to install the boilerplate using just the following command.

```bash
composer create-project --prefer-dist langleyfoxall/laravel-boilerplate
```

**Comment by DivineOmega** [comment]
Now released on Packagist, which closes this issue!

https://packagist.org/packages/langleyfoxall/laravel-boilerplate

**Comment by DivineOmega** [comment]
Re-opening this, as we should probably document the new installation procedure as part of the README.

**Comment by DivineOmega** [comment]
Fixed in #18 and #21.
